eox_nelp/api_clients/pearson_rti.py

```diff
@@ -0,0 +1,144 @@
+"""Client module for Pearson VUE RTI service.
+
+Classes:
+    PearsonRTIApiClient: Class to interact with the pearson vue Real-Time Import external service.
+"""
+import logging
+
+from django.conf import settings
+
+from eox_nelp.api_clients import AbstractSOAPClient
+from eox_nelp.api_clients.authenticators import PKCS12Authenticator
+
+try:
+    from eox_audit_model.decorators import audit_method
+except ImportError:
+    def audit_method(action):  # pylint: disable=unused-argument
+        """Identity audit_method"""
+        return lambda x: x
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class PearsonRTIApiClient(AbstractSOAPClient):
+    """This class implements the required method to consume APIs available in the Pearson VUE systems
+    for importing candidate demographic and exam authorization data.
+    """
+    authentication_class = PKCS12Authenticator
+
+    def __init__(self):
+        self.cert = getattr(settings, "PEARSON_RTI_CERT")
+        self.passphrase = getattr(settings, "PEARSON_RTI_PASSPHRASE")
+
+        super().__init__()
+
+    @property
+    def base_url(self):
+        return getattr(settings, "PEARSON_RTI_WSDL_URL")
+
+    def ping(self, payload):
+        """The Ping service allows to ping the VUE system to verify that web services are available.
+        This allows to validate the credentials(ping_database) if they are included into the payload.
+
+        Arguments:
+            payload<str>: Body xml request in string format.
+
+        Returns:
+            <Dict>: Parsed xml response to Dict format.
```

Comment:
This is a reason you use this shape of dict ?
I mean the result in all tests has keys for each case. But do you want to save only the selected keys?

Reply:
yes, I don't  see the point of keeping fields like `soap:Envelope` since that doesn't affect the main purpose of this